app/presenters/notices/setup/notice-type.presenter.js

```diff
@@ -23,43 +25,52 @@ function go(session) {
 }
 
 function _backLink(sessionId, checkPageVisited, journey) {
-  if (journey === 'standard') {
-    return `/system/notices`
+  if (journey === NoticeJourney.STANDARD) {
+    return {
+      href: `/system/notices`,
+      text: 'Back'
+    }
   }
 
   if (checkPageVisited) {
-    return `/system/notices/setup/${sessionId}/check-notice-type`
+    return {
+      href: `/system/notices/setup/${sessionId}/check-notice-type`,
+      text: 'Back'
+    }
   }
 
-  return `/system/notices/setup/${sessionId}/licence`
+  return {
+    href: `/system/notices/setup/${sessionId}/licence`,
+    text: 'Back'
+  }
 }
 
 function _options(noticeType, journey) {
-  if (journey === 'standard') {
+  if (journey === NoticeJourney.STANDARD) {
     return [
       {
-        checked: noticeType === 'invitations',
-        value: 'invitations',
+        checked: noticeType === NoticeType.INVITATIONS,
+        value: NoticeType.INVITATIONS,
         text: 'Returns invitation'
       },
       {
-        checked: noticeType === 'reminders',
-        value: 'reminders',
+        checked: noticeType === NoticeType.REMINDERS,
+        value: NoticeType.REMINDERS,
         text: 'Returns reminder'
       }
     ]
   }
 
   return [
     {
-      checked: noticeType === 'invitations',
-      value: 'invitations',
+      checked: noticeType === NoticeType.INVITATIONS,
+      value: NoticeType.INVITATIONS,
       text: 'Standard returns invitation'
     },
     {
-      checked: noticeType === 'returnForms',
-      value: 'returnForms',
-      text: 'Submit using a paper form invitation'
+      checked: noticeType === NoticeType.PAPER_RETURN,
+      value: NoticeType.PAPER_RETURN,
+      text: 'Submit using a paper return invitation'
```

Comment:
Just needs to be 'Paper return'.

And although not part of this, 'Standard returns invitation' needs to be updated to 'Returns invitation'.

I'm happy for that change to be included as part of this.